Source/Core/Core/PowerPC/JitInterface.cpp

```diff
@@ -98,6 +98,30 @@ void JitInterface::SetProfilingState(ProfilingState state)
   m_jit->jo.profile_blocks = state == ProfilingState::Enabled;
 }
 
+void JitInterface::UpdateMembase()
+{
+  auto& ppc_state = m_system.GetPPCState();
+  auto& memory = m_system.GetMemory();
+  if (ppc_state.msr.DR)
+  {
+#ifdef _M_X86_64
+    ppc_state.mem_ptr = memory.GetLogicalBase();
+#endif
+#ifdef _M_ARM_64
+    ppc_state.mem_ptr = m_jit->jo.fastmem_arena ? memory.GetLogicalBase() : memory.GetLogicalPageMappingsBase();
+#endif
```

Comment:
You could probably just use the AArch64 variant of this on all CPUs. Jit64 shouldn't rely on RMEM containing the logical or physical base when there is no fastmem_arena, since in that case there is no logical or physical base.